### PR TITLE
add sys/backup endpoint

### PR DESCRIPTION
**Pull request by chrismerck**

### CODE REVIEW

File: common/paths.yaml [hunk 1]
```diff
@@ -172,7 +172,7 @@
 /v2/sys/free:
   get:
     $ref: ../firmware/paths.yaml#/FreeGet
-/v2/sys/backup:
+/v2/sys/snapshot:
```
Comment (endy-s) [inline common/paths.yaml:175]: Looks like it was implemented as `backup` actually
```suggestion
/v2/sys/backup:
```

File: firmware/schemas.yaml [hunk 2]
```diff
@@ -265,29 +297,14 @@ Backup:
       readOnly: true
       description: |
         Number from 0-1000 representing progress of `backup` or `restore`.
+        Progress == 1000 iff backup == 2 or restore == 2.
```
Comment (endy-s) [inline firmware/schemas.yaml:300]: Typo
```suggestion
        Progress == 1000 if backup == 2 or restore == 2.
```

**Review by endy-s — CHANGES_REQUESTED** [review]
The path change is critical

File: firmware/schemas.yaml (hunk 2 above)
Comment (chrismerck) [reply firmware/schemas.yaml:300]: Actually, was not a typo. "Iff" stands for "if and only if".

https://en.wikipedia.org/wiki/If_and_only_if

**Pull request merged**